_posts/2023-02-26-pernStack.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 1st Full Stack Webapp with PERN Stack
+layout: post
+banner_image: https://images.unsplash.com/photo-1522202176988-66273c2fd55f?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1471&q=80
+---
+
+The team first full stack application journey
+
+My FE team is tasked with creating a simple full stack application. We decided to use PERN stack because our knowledge of Javascript which would covered 3 out of 4 tech in the stack, with combination of these tech allowed us to build a full stack web application with CRUD operations.
```

Comment:
> My FE team *was* tasked...


You define PERN later, but you should at least put in what the acronym stands for when you first reference it:
> We decided to use Postgres, Express, React, and Node (PERN stack) because...

The second sentence is really long and probably should be broken up:
> We decided to use PERN stack because of our knowledge of Javascript would cover three out of four technologies in the stack.  The combination of these technologies allowed us to build...

Reply:
If you are going to highlight why you used the PERN stack here, I'd leave those reasons out from down below.